src/components/Card/Card.js

```diff
@@ -51,6 +51,26 @@ const LikeButton = ({ onClick, liked }) => (
   </button>
 );
 
+const Info = ({mentor}) => {
```

Comment:
Nice! I always advocate for creating smaller components, thanks!

Reply:
Me too! It's easier to maintain smaller components than hunting a bug in a component made up of 1000 lines.

Reply:
I like this as well!